src/pydocstyle/data/imperatives_blacklist.txt

```diff
@@ -0,0 +1,109 @@
+# Blacklisted imperative words
+#
+# These are words that, if they begin a docstring, are a good indicator that
+# the docstring is not written in an imperative voice.
+#
+# The words included in this list fall into a number of categories:
+#
+# - Starting with a noun/pronoun indicates that the docstring is a noun phrase
+#   or a sentence but not in the imperative mood
+# - Adjectives are always followed by a noun, so same
+# - Particles are also followed by a noun
+# - Some adverbs don't really indicate an imperative sentence, for example
+#   "importantly" or "currently".
+# - Some irregular verb forms that don't stem to the same string as the
+#   imperative does (eg. 'does')
+:param
+:return
+:return:
+:returns
+:returns:
+:rtype
```

Comment:
Is in necessary to put the colons here? We can just check for that when checking for D401.